Dockerfile.rhel

```diff
@@ -12,8 +12,8 @@ COPY .htaccess README.md *.sh /build/
 COPY /plugins /build/plugins
 COPY /v2 /build/v2
 WORKDIR /build/
-RUN cd plugins && ./check_plugins_location_v1.sh
-RUN cd v2/plugins && ./check_plugins_location_v2.sh
+RUN ./check_plugins_location_v1.sh
+RUN ./check_plugins_location_v2.sh
```

Comment:
My bad, sorry. Can you also check whether it is needed to add `mkdir v2` where we do `mkdir plugins`?

Reply:
37th line

Reply:
no worries, well original Dockerfile is built just fine and does not contain `mkdir v2` but I have not tried to deploy in on minishift and verify that all is hosted fine. Should I ?

Reply:
It contains `RUN mkdir /var/www/html/plugins` so I'm wondering whether we should add the same for v2

Reply:
But this doesn't block this PR at all

Reply:
Ah, got it so you think both rhel and non-rhel might need to be updated with this fix `RUN mkdir /var/www/html/plugins/v2` since v2 might not be hosted without this change?

Reply:
Non-rhel image works for me
Not sure about Rhel one

Reply:
ok. let's merge and see ;-)